ksqldb-streams/src/main/java/io/confluent/ksql/execution/streams/SourceBuilder.java

```diff
@@ -221,6 +233,28 @@ private static PhysicalSchema getPhysicalSchemaWithKeyAndPseudoCols(
     );
   }
 
+  private static PhysicalSchema getPhysicalSchemaWithPseudoColumnsToMaterialize(
```

Comment:
This method is basically identical to `getPhysicalSchemaWithKeyAndPseudoCols()`. Can we de-dup?